modules/client-configuration/pages/clients-alibaba.adoc

```diff
@@ -0,0 +1,129 @@
+[[clients-alibaba]]
+= Registering {alibabaclo} Clients
+
+This section contains information about registering traditional and Salt clients running {alibabaclo} operating systems.
+
+ifeval::[{suma-content} == true]
+[IMPORTANT]
+====
+{alibabaclo} repository URLs are available from {scc}, but the packages and metadata are provided by Alibaba Cloud, not by {suse}.
+====
+endif::[]
+
+
+[IMPORTANT]
+====
+The traditional stack is available on {alibabaclo}{nbsp}2 but it is not supported.
+{alibabaclo}{nbsp}2 clients are only supported as Salt clients.
+====
+
+[WARNING]
+====
+Some {alibabaclo}{nbsp}2 instances will need two tries to register successfully.
+====
+
+== Add Software Channels
+
+Before you register {alibabaclo} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+ifeval::[{suma-content} == true]
+
+The products you need for this procedure are:
+
+[[alibaba-channels-wizard]]
+[cols="1,1", options="header"]
+.Alibaba Cloud Linux Products - WebUI
+|===
+| OS Version | Product Name
+| {alibabaclo} 2 | Alibaba Cloud Linux 2 x86_64
+|===
+
+include::snippets/addchannels_vendor_webui.adoc[]
+
+Alternatively, you can add channels at the command prompt.
+The channels you need for this procedure are:
+
+[[alibaba-channels-suma-cli]]
+[cols="1,1", options="header"]
+.Alibaba Cloud Linux Channels - CLI
+|===
+| OS Version | Base Channel
+| {alibabaclo} 2 | alibaba-2-x86_64
+|===
+
+include::snippets/addchannels_vendor_cli.adoc[]
+
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+The channels you need for this procedure are:
+
+[[alibaba-channels-uyuni-cli]]
+[cols="1,1,1", options="header"]
```

Comment:
I think you need "1,1,1,1" here, as otherwise the table does not render correctly:

![image](https://user-images.githubusercontent.com/4226070/113988279-9a177900-984f-11eb-97e9-36587215fecf.png)

Reply:
Was this fixed? Think it's still pending?

Reply:
I just added the extra column. We should be ok now.